src/oss/confluent.sh

```diff
@@ -1741,6 +1741,13 @@ invalid_requirement() {
 
 requirements
 
+cat <<EOF
+Note: This CLI is not intended for production use.
+For production environment documentation, please refer to:
+https://docs.confluent.io/current/installation/installing_cp.html
```

Comment:
I decided against linking this explicitly, since the link I posted has a section called "Production Environments" which references the script/systemd page you've referenced. I don't feel strongly, though, and would be happy to put both, noting that if this message gets printed command, I was hoping to keep it succinct.

Reply:
Agree that we don't need two links -- we just need one good one.  In thinking about it, I think your link is good enough for now.  In the future, we may get a better link (DOCS-383, DOCS-426)

Reply:
@cyrusv actually how about this idea.  Instead of talking about it from production point of view:

```
This CLI is not intended for production use.
 For production environment documentation, please refer to:
 https://docs.confluent.io/current/installation/installing_cp.html
 ```

How about talking about it from dev point of view and link to Confluent CLI doc...

```
This CLI is intended for development only, not for production
https://docs.confluent.io/current/cli/index.html
```

Benefits of doing it this way:
1. Point users to explore/discover more about Confluent CLI
2. It is easier to iterate on updating docs as needed, with pointers to production docs as those docs get updated, than to iterate on updating the tool with the right links